pkg/lib/session/session.go

```diff
@@ -15,6 +15,17 @@ const (
 	TypeOfflineGrant     Type = "offline_grant"
 )
 
+type Session interface {
```

Comment:
Do not use the original name. if we use a new name, other PR that also refer to Session (the old one) will fail to compile. That helps us to spot problems easier.

Let's use `ResovledSession`